static/load_moc.ts

```diff
@@ -7,22 +7,16 @@ async function addPackage(name, repo, version, dir) {
   const base_url = `https://cdn.jsdelivr.net/gh/${repo}@${version}`;
   const response = await fetch(meta_url);
   const json = await response.json();
-  const promises = [];
   const fetchedFiles = [];
-  for (const f of json.files) {
+  await Promise.all(json.files.map(async f => {
     if (f.name.startsWith(`/${dir}/`) && /\.mo$/.test(f.name)) {
-      const promise = (async () => {
-        const content = await (await fetch(base_url + f.name)).text();
-        const stripped = name + f.name.slice(dir.length + 1);
-        fetchedFiles.push(stripped);
-        Motoko.saveFile(stripped, content);
-      })();
-      promises.push(promise);
+      const content = await (await fetch(base_url + f.name)).text();
+      const stripped = name + f.name.slice(dir.length + 1);
+      fetchedFiles.push(stripped);
+      await Motoko.saveFile(stripped, content);
```

Comment:
Do we know if `Motoko` module methods are all async calls? I was under the impression that they are synchronous?

Reply:
I added the "await" keyword in case these calls become asynchronous due to using a service worker or anything similar (as in the case of Motoko Playground).

Reply:
Okay, so it's harmless to put `await` anywhere, right?

Reply:
Yep, as long as the return value isn't expected to be a "[thenable](https://www.stefanjudis.com/today-i-learned/await-in-async-functions-works-for-any-thenable/)."